### DISCUSSION TITLE
[功能建议]强烈建议增加对maridb的兼容

### DISCUSSION THREAD

**Discussion by l0cugoo** [hhyo/Archery#1768] [General]
使用archery.oracle一段时间，很不错，
当前有使用maridb，建议支持
ps：目前是可以通过mysql链接方式强连maridb做查询哦

**Comment by hhyo** [comment]
请问下当前maridb有哪些场景不能兼容？

**Reply by l0cugoo** [reply]
感谢回复

---------------------------
1、直接问题（检测sql(DML)报错）,细节如下：

1.1、具体问题：SQL审核->SQL上线->SQL检测 报错：
GoInception检测语句报错，请注意检查系统配置中GoInception配置，错误信息
connect() argument 1 must be str, not None
1.2、基础配置：maridb数据库，用mysql链接方式配置

2、间接问题：
2.1、源码中 配置文件 Archery/sql/engines/mysql.py里没有maridb的配置文件
2.2、页面：首页›Sql›实例配置›增加实例配置 中的[数据库类型]没有maridb类型

预期：
Archery中有专门的maridb配置，并且sql上线正常

**Reply by l0cugoo** [reply]
![image](https://user-images.githubusercontent.com/28584300/185830743-c9b36fb0-a699-4308-b424-de8a0a103570.png)

**Reply by hhyo** [reply]
错误很清楚，检查相关配置项，用做审核和查询必须要有该配置

可以配置为mysql类型，看具体有什么场景不支持，也方便这边进行适配，理论上都支持
